packages/hawtio/src/plugins/shared/util/helpers.ts

```diff
@@ -0,0 +1,14 @@
+/**
+ * Will format a property to a standart human readable string with its spaces.
+ * It will respect MBean and leave it together
+ * @param str The property to transform
+ * @returns The property with its proper spaces
+ */
+export const tidyLabels = (str: string) =>
+  str
+    .replace(/([a-z])([A-Z])/g, '$1 $2')
+    .replace(/\b([A-Z]+)([A-Z])([a-z])/, '$1 $2$3')
+    .replace('M Bean', 'MBean')
+    .replace(/^./, function (str) {
+      return str.toUpperCase()
+    })
```

Comment:
also, we use `humanizeXxx` for performing the same thing in the old verison. I don't see reasons dare to use the other name here. Changing the name makes it a bit hard to map between the new and old impl in this case, IMO.

Reply:
Agreed. Will change it